src/App.tsx

```diff
@@ -1,25 +1,55 @@
-import React from 'react';
+import { useState } from 'react';
 import './App.css';
+import { Pagination } from './components/Pagination';
 import { getNumbers } from './utils';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42).map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [itemsPerPage, setItemsPerPage] = useState('5');
+  const [currentPage, setCurrentPage] = useState(1);
+  const [startItem, setStartItem] = useState(1);
```

Comment:
why do you need this state, it looks like currentPage and startItem are equal 

I think you can remove startItem state at all